src/change_stream/event.rs

```diff
@@ -114,10 +114,25 @@ pub struct UpdateDescription {
 
     /// An array of field names that were removed from the `Document`.
     pub removed_fields: Vec<String>,
+
+    /// Arrays that were truncated in the `Document`.
+    pub truncated_arrays: Vec<TruncatedArray>,
+}
+
+/// Describes an array that has been truncated.
+#[derive(Debug, Serialize, Deserialize, PartialEq)]
+#[serde(rename_all = "camelCase")]
+#[non_exhaustive]
+pub struct TruncatedArray {
```

Comment:
Is there a ticket for adding support for this (from a DRIVERS one) that we ought to close out?

Reply:
It looks like this was https://jira.mongodb.org/browse/RUST-659, which was closed out as "Will handle in our initial implementation of Change Streams."